app/frontend/components/ConcertoYoutubeVideo.vue

```diff
@@ -15,13 +15,30 @@ const videoUrl = computed(() => {
   return `https://www.youtube-nocookie.com/embed/${props.content.video_id}?rel=0&iv_load_policy=3&autoplay=1&controls=0&playsinline=1&mute=1&enablejsapi=1`;
 })
 
+const playerRef = ref(null);
+let player = null;
+
 function isYTAPILoaded() {
   /* global YT */
   return (window.YT && window.YT.Player);
 }
 
 async function loadYTAPI() {
   return new Promise((resolve) => {
+    // Check if script is already in the DOM
+    const existingScript = document.querySelector('script[src="https://www.youtube.com/iframe_api"]');
```

Comment:
LIke above, the `https://player.vimeo.com/api/player.js` string is duplicated and should be DRYed up.